### PR TITLE
Semantic fixes

### PR DESCRIPTION
**Pull request by mgian**
Some updates to remove some warnings in QtCreator and to move forward to more modern code

### CODE REVIEW

**Comment by GillesDebunne** [comment]
Thanks a lot for this PR. I'll review it as soon as I can.

File: QGLViewer/qglviewer.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include <QGL>
 #include <QMap>
 #include <QTime>
+#include <QGL>
```
Comment (GillesDebunne) [inline QGLViewer/qglviewer.h]: Done above
```suggestion
```

File: examples/overpainting/viewer.cpp [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #include <QGL>
 #include <QPainter>
+#include <QGL>
```
Comment (GillesDebunne) [inline examples/overpainting/viewer.cpp]:
```suggestion
```

**Comment by mgian** [comment]
I found that I made the PR starting from the 2.7.0 version.
I will update my fork to the last version and will what happen

**Pull request closed without merging**